docs/proposals/20220725-managed-kubernetes.md

```diff
@@ -435,11 +442,9 @@ The reasons for this recommendation are as follows:
 - The infra cluster provisions and manages the general infrastructure required for the cluster but not the control plane.
 - By having a separate infra cluster API definition, it allows differences in the API between managed and unmanaged clusters.
 
-Providers like CAPZ and CAPA have already implemented managed Kubernetes support and there should be no requirement on them to move to Option 3. Both Options 2 and 4 are solutions that would work with ClusterClass and so could be used if required.
-
-Option 1 is the only option that will not work with ClusterClass and would require a change to CAPI. Therefore this option is not recommended.
+Providers like CAPZ and CAPA have already implemented managed Kubernetes support and there should be no requirement on them to move to Option 3. Both Options 4 also works well with ClusterClass and so could be used if required. Option 2 will no longer be relevant once CAEP https://github.com/kubernetes-sigs/cluster-api/pull/8500 is implemented.
 
-*** This means that CAPA will have to make changes to move away from Option 1 if it wants to support ClusterClass.
+Option 1 will be available when CAEP https://github.com/kubernetes-sigs/cluster-api/pull/8500 is implemented. Until then it is not recommended.
 
 ### Additional notes on option 3
```

Comment:
We should also update the recommendations section slightly.

For new implementations:

If you have any extra infra then option 3 (with ability to report apiserver endpoint back via CP) remains the main recommendation as its in line with the general CAPI recommendations.

If you have no extra infra then option 1 (assuming it's edited based on a previous comment).

For historic implementations (i.e. CAPA, CAPZ), there is still no requirement to move to another implementation.

Reply:
Thinking about it, @jackfrancis how would you feel about me @Winnie making the updates to the managed k8s proposal? We can make them on your branch?

Reply:
I've made some changes in response to feedback but I'll step away for a day or so, feel free to update the original doc in my branch here!

After that let's circle back and figure out how to address your comments about how we're describing the "once [this CAEP](https://github.com/kubernetes-sigs/cluster-api/pull/8500) is implemented..." remarks.

Reply:
@jackfrancis - sorry i've been a bit hands off for the last few weeks.....my head has been elsewhere with some work stuff.